--- a/setup.py	2020-09-09 16:39:47.000000000 +0000
+++ b/setup.py	2021-03-22 11:35:31.171587290 +0000
@@ -25,7 +25,6 @@
                      ('HAVE_SYS_SYSINFO_H', '1'),
                      ('_FILE_OFFSET_BITS', '64')]
     libraries = ['crypto', 'rt']
-    includes = ['/usr/local/include', '/usr/include']
     CFLAGS.append('-O2')
 elif sys.platform.startswith('win32'):
     define_macros = [('inline', '__inline')]
